app/assets/stylesheets/components/_event.sass

```diff
@@ -0,0 +1,24 @@
+$Event-backgroundColor: Theme-color(background, default)
+$Event-borderTop-color: #FF623F  // FIXME Put this color on background color MAP on theme.sass file.
```

Comment:
Yes, you are right. From generic to specific, left to right...
It could be. only the backgroundColor is the css property name, maybe it make more sense to have it `Event-backgroundColor`. Or we can have always the template `Component-color-terget`. Either way, it must be consistent. Should we go with the second, Event-color-background?